src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -239,10 +240,21 @@ private void ListView_SizeChanged(object sender, SizeChangedEventArgs e)
             // adjust the width of the "project" column so that it takes
             // up all remaining width.
             var gridView = (GridView)_projectList.View;
-            var width = _projectList.ActualWidth - 3 * SystemParameters.VerticalScrollBarWidth;
+            var width = _projectList.ActualWidth - 8;
+
+            // If there are more than 7 projects the scrollbar will be visible
+            if (_projectList.Items.Count > 7)
```

Comment:
Could the screenshots from testing be added to this PR so we can see how this works with different resolutions/etc? I'm not sure if we need PLOC testing, but maybe ensuring the minimum resolution supported looks as expected would be good enough.

Reply:
![image](https://github.com/user-attachments/assets/3f12b515-487b-4841-aa92-9be25d8d10d3)

Reply:
Above image is how it looks with a low resolution. Details pane has a minimum width so windows size doesn't really affect the table content.

Reply:
Great! I think this should be in the PR description as well.

Reply:
https://github.com/NuGet/NuGet.Client/pull/6051
Realized that there was missing localization and narrator

Reply:
Done! Added this and accessibility testing to the PR description